packages/xo-web/src/common/xo/index.js

```diff
@@ -446,9 +446,9 @@ const missingPatchesByHost = {}
 export const subscribeHostMissingPatches = (host, cb) => {
   const hostId = resolveId(host)
 
-  if (missingPatchesByHost[hostId] == null) {
-    missingPatchesByHost[hostId] = createSubscription(() => getHostMissingPatches(host))
-  }
+  host.power_state !== 'Running'
+    ? (missingPatchesByHost[hostId] = () => noop)
```

Comment:
Only subscriptions can be stored in the collection `missingPatchesByHost`, otherwise, it will break this method https://github.com/vatesfr/xen-orchestra/blob/dd19e5548a29abb70ea3b0f76e9a520ccc733a16/packages/xo-web/src/common/xo/index.js#L455-L465